hello_world_on_chain/README.md

```diff
@@ -243,7 +217,7 @@ which show that the [`User`](contracts/User.sol) contract was able to
 verify the TA in just over 2 seconds.
 If you go to Basescan to see 
 [contract transaction logs](https://sepolia.basescan.org/tx/0x82eb1d2cd500a1e236678a04f9ee8361e2fda11224d0791612ae2244d294bf78#eventlog)
-you can see that the `verifyAttestedFnCallClaims` emitted the
+you can see that the `processTAHelloWorld` emitted the
```

Comment:
@mwittie since I don't have creds to run this, I did not yet update "Step 

Some updates that are definitely needed (there may be others that cascade):
1. some of the outputs have changed.
3. the contract has changed so the contract needs to be redeployed.
4. the value in `deployments/user_deployed_address` needs to be updated.
5. the link in the block explorer needs to be updated.

Reply:
udpated